README.md

```diff
@@ -10,9 +10,6 @@ Omes (pronounced oh-mess) is the Hebrew word for "load" (עומס).
 ## Prerequisites
 
 - [Go](https://golang.org/) 1.24+
-  - `protoc` and `protoc-gen-go` must be installed
-  - tip: don't worry about the specific versions here; instead, the GitHub action will make a diff
-    available for download that you can use with `git apply`
```

Comment:
Might be worth changing this to say you _either_ need proto or docker to get kitchen sink going locally